### PR TITLE
Merge Hotfix v1.11.1 after release

### PR DESCRIPTION
**Pull request by mwithi**

### CODE REVIEW

File: src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java [hunk 1]
```diff
@@ -249,18 +249,17 @@ protected void validateAdmission(Admission admission, boolean insert) throws OHS
...
 		}
 		LocalDateTime dateIn = admission.getAdmDate();
 		if (dateIn == null) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.admission.admissiondatecannotbeempty.msg"),
-					OHSeverityLevel.ERROR));
+							MessageBundle.getMessage("angal.admission.admissiondatecannotbeempty.msg"),
+							OHSeverityLevel.ERROR));
 			throw new OHDataValidationException(errors);
 		}
```
Comment (mwithi) [inline src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java:262]:
```suggestion
		}
		if (dateIn.isAfter(today)) {
```

File: src/main/java/org/isf/dicom/service/DicomIoOperationRepository.java [hunk 2]
```diff
@@ -49,6 +49,9 @@
 	@Transactional
 	@Query("delete from FileDicom fd WHERE fd.patId = :id AND fd.dicomSeriesNumber = :file")
 	void deleteByIdAndNumber(@Param("id") int id, @Param("file") String file);
+    
+    @Query(value = "SELECT COUNT(DM_FILE_SER_NUMBER) FROM DICOM WHERE DM_FILE_SER_NUMBER = :dicomSeriesNumber", nativeQuery= true)
+	int seriesExists(@Param("dicomSeriesNumber") String dicomSeriesNumber);
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/service/DicomIoOperationRepository.java:54]:
```suggestion
```

File: src/main/java/org/isf/dicom/model/FileDicom.java [hunk 3]
```diff
@@ -50,6 +50,8 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
+import com.drew.lang.annotations.Nullable;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/dicom/model/FileDicom.java:53]: This import is not needed and is unused.

**Comment by dbmalkovsky** [comment]
If my two comments are addressed then it LGTM.

File: src/main/java/org/isf/visits/manager/VisitManager.java [hunk 4]
```diff
@@ -181,7 +181,7 @@ public void deleteVisit(Visit visit) {
 	 */
 	@Transactional(rollbackFor = OHServiceException.class)
 	public boolean newVisits(List<Visit> visits) throws OHServiceException {
-		return newVisits(visits, new ArrayList<>());
+		return newVisits(visits, new ArrayList<Visit>());
```
[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/visits/manager/VisitManager.java:184]: true! thanks!

Comment (mwithi) [inline src/main/java/org/isf/visits/manager/VisitManager.java:184]:
```suggestion
		return newVisits(visits, new ArrayList<>());
```

File: src/main/java/org/isf/dicom/model/FileDicom.java (hunk 3 above)
Comment (mwithi) [inline src/main/java/org/isf/dicom/model/FileDicom.java:53]:
```suggestion
```

Comment (mwithi) [reply src/main/java/org/isf/dicom/model/FileDicom.java:53]: Thanks!

File: src/main/java/org/isf/visits/manager/VisitManager.java (hunk 4 above)
Comment (mwithi) [inline src/main/java/org/isf/visits/manager/VisitManager.java]:
```suggestion
		return newVisits(visits, new ArrayList<>());
```

File: src/main/java/org/isf/dicom/model/FileDicom.java [hunk 5]
```diff
@@ -50,6 +50,7 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
+
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/model/FileDicom.java:53]:
```suggestion

```

**Pull request merged**